### DISCUSSION TITLE
desktop app port / PulseView compatibiliy

### DISCUSSION THREAD

**Discussion by flxzt** [fhdm-dev/scoppy#72] [General]
Hi, first: I love the project, it is so well done.

Are there any plans for a port to desktop, especially linux?
The small screens of my phone just does not work all that well for me unfortunately.
Or, if it is much effort, I think it should be possible to interface with sigrok / pulseview.

**Comment by fhdm-dev** [comment]
Hi,

Thanks for your interest in the project.

I don't have any immediate plans to port the app to the desktop but it might happen sometime in the future. And I'm not sure how much work would be required to interface with sigok. I'd have to look into it.

The focus at the moment is adding more features to the Android app and the firmware, especially WiFi support.
